client/my-sites/sidebar/style.scss

```diff
@@ -0,0 +1,21 @@
+.sidebar__sparkline {
+	width: 80px;
+	height: 24px;
+	position: absolute;
+	right: 16px;
+	top: 8px;
+
+	@include breakpoint( "<660px" ) {
+		top: 13px;
```

Comment:
The video isn't working for me, but I'll test it out - thanks for catching that.